hawp/detector.py

```diff
@@ -155,10 +159,13 @@ def forward(self, images, annotations = None):
             v2e_idx1 = v2e_dis1.argmin(dim=0)
             v2e_idx2 = v2e_dis2.argmin(dim=0)
             edge_indices = torch.stack((v2e_idx1,v2e_idx2)).t()
-            wireframe = WireframeGraph(juncs_final,juncs_score,edge_indices,score_final,output.size(3),output.size(2))
 
-            wireframe.rescale(annotations[0]['width'],annotations[0]['height'])
-        
+            if not self.export_mode:
+                wireframe = WireframeGraph(juncs_final,juncs_score,edge_indices,score_final,output.size(3),output.size(2))
+                wireframe.rescale(annotations[0]['width'],annotations[0]['height'])
+            else:
+                return juncs_final,juncs_score,edge_indices,score_final,output.size(3),output.size(2)
```

Comment:
output.size(3) and output.size(2) are not tensors anymore, they probably won't have meaningful data once exported